examples/features/torch_batch_process/batch_inference/compare_with_core_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Batch inference with Core API & Torch Batch Processing API
+## Overview
```

Comment:
Let's make sure to have empty lines above and below markdown headers (also below).

Well.. not above the very toplevel header, of course.

But it makes it easier to read the markdown source, and is a pretty standard convention in markdown source files.  Looks like it's not required by the markdown renderer though, so this is more of a nit.